README.md

```diff
@@ -394,6 +394,7 @@ Simple deployment of [E-mail](https://en.wikipedia.org/wiki/Email) servers, e.g.
 - [Dovecot](https://www.dovecot.org/) - IMAP and POP3 server written primarily with security in mind. ([Source Code](https://github.com/dovecot/core)) `MIT/LGPL-2.1` `C`
 - [MailForm](https://github.com/Feuerhamster/mailform) - Lightweight self-hosted open source alternative to Formspree and SendGrid. `Apache-2.0` `Nodejs`
 - [Piler](https://www.mailpiler.org/wiki/start) - Feature-rich open source email archiving solution. ([Source Code](https://bitbucket.org/jsuto/piler)) `GPL-3.0` `C`
+- [Stalwart JMAP](https://stalw.art/jmap) - JMAP and IMAP server designed to be secure, fast, robust and scalable. ([Source Code](https://github.com/stalwartlabs/jmap-server)) `AGPL-3.0` `Rust`
```

Comment:
Thank you for reviewing the PR. I am not sure I understand the change. The software was written in Rust. Docker is just one of the ways the software can be run. 
On the product documentation there are details to run the software both with or without Docker.

Reply:
`The Language tag is the main server-side requirement for the software. Don't include frameworks or specific dialects.`

Although its written in rust, rust isn't needed for the software, if it is, I couldn't find it. 

If rust is required for the software, then please change the language to `Rust/Docker`. If not then please change it too `Docker`

Reply:
> If rust is required for the software, then please change the language to Rust/Docker. If not then please change it too Docker

Well I disagree, https://jmap.stalw.art/install.sh -> It downloads and installs a single, static binary and a few configuration files, there are no major requirements aside from an operating system and libc, in this case we use to set the language/platform to `Rust` because that's the way most Rust programs are distributed (the same goes for Go, C, ...).

So `Rust` or `Rust/Docker` is correct.